api/src/main/java/org/openmrs/module/registrationcore/RegistrationCoreUtil.java

```diff
@@ -45,6 +52,23 @@ else if (days == null || days == 0){
 
         return c.getTime();
     }
-
-
+    
+    /**
+     * Get the month from which the birth dates are estimated, defaults to January
+     *
+     * @return
+     */
+    public static int getEstimationStartMonth() {
+        int startMonth = Calendar.JANUARY;
+        String gpStartMonth = Context.getAdministrationService().getGlobalProperty(RegistrationCoreConstants.GP_BIRTHDATE_ESTIMATION_START_MONTH);
```

Comment:
@dkayiwa I am confused - doesn't the logic of the method do that - since the startMonth has  a default value? We can never be sure if the GP exists